lib/ansible/plugins/lookup/nios_next_network.py

```diff
@@ -0,0 +1,102 @@
+#
+#  Copyright 2018 Red Hat | Ansible
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+lookup: nios_next_network
+version_added: ""
+short_description: Return the next available network for a network
+description:
+  - Uses the Infoblox WAPI API to return the next available network addresses
+    for a given network CIDR
+requirements:
+  - infoblox_client
+extends_documentation_fragment: nios
+options:
+    _terms:
+      description: The CIDR network to retrieve the next network from
+      required: True
+    cidr:
+      description: The CIDR of the network to retrieve the next network from
+      required: True
+    num:
+      description: The number of IP addresses to return
```

Comment:
@dionben please change the description statement according to the network search.